### PR TITLE
adds healthy and unhealthy transformations

### PR DESCRIPTION
**Pull request by jholm117**

### CODE REVIEW

File: project2/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 This project uses [`pipenv`](https://pipenv.readthedocs.io/en/latest/) to manage dependencies. Make sure you have it installed before working on the project:
 
 ```sh
-pip install --user pipenv
+pip install pipenv
```
Comment (jholm117) [inline project2/README.md:8]: `--user` flag was giving people issues so may as well just not deal with this

File: project2/README.md [hunk 2]
```diff
@@ -53,14 +53,14 @@ pipenv run lint
 `py/main.py` takes an AllRecipes url from standard input and prints a transformed recipe to standard ouput. Eg:
 
 ```sh
-python py/main.py <transform_code> < address
+python py/main.py {transform_code} < address
```
Comment (jholm117) [inline project2/README.md:56]: for clarity.

people were missing the `<` operator so hopefully this is more contrast

File: project2/address [hunk 3]
```diff
@@ -1 +1 @@
-https://www.allrecipes.com/recipe/10813/best-chocolate-chip-cookies/?internalSource=hub%20recipe&referringContentType=Search
+https://www.allrecipes.com/recipe/230900/brown-rice-curry/?internalSource=hub%20recipe&referringContentType=Search
```
Comment (jholm117) [inline project2/address:1]: this was more useful than chocolate chip cookies for me

File: project2/README.md [hunk 4]
```diff
@@ -53,14 +53,14 @@ pipenv run lint
...
 
 - `veg` - to vegetarian
-- `meat*` - from vegetarian
-- `healthy*` - to healthy
-- `unhealthy*` - from healthy
+- `meat` - from vegetarian
+- `healthy` - to healthy
+- `unhealthy` - from healthy
 
 \* not yet supported
```
Comment (MagicWilliams) [inline project2/README.md]: We can remove the not yet supported comment now, right?

File: project2/py/transform.py [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+    """
+        scales quantity of ingredient by factor
+    """
+    new_quantity = ingredient.quantity * factor
+    new_ingredient = ingredient.substitute_quantity(new_quantity)
+    return new_ingredient
+
+
+def change_quantities(recipe, predicate, factor):
+    """
```
Comment (MagicWilliams) [inline project2/py/transform.py:71]: are these comments too or strings?

File: project2/py/transform.py [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+        scale quantity of all ingredients in a recipe that satisfy predicate by factor
+    """
```
Comment (MagicWilliams) [inline project2/py/transform.py:73]: same here ^^

**Review by MagicWilliams — COMMENTED** [review]
minor comment nits, otherwise code appears to be good to me

i checked out the branch though, it appears this new NLTK import is getting in the way of me actually being able to run this branch on my machine

if everything works, tthen bombs away & hopefully this is just an installation issue on my end

File: project2/README.md (hunk 4 above)
Comment (jholm117) [reply project2/README.md]: yessir

File: project2/py/transform.py (hunk 5 above)
Comment (jholm117) [reply project2/py/transform.py:71]: this is a python multi-line comment, they're convention for function documentation in python

**Pull request merged**